build.win32x86/common/Makefile

```diff
@@ -250,24 +260,25 @@ $(VMDEF) $(VMEXP) $(VMLIB): $(VMOBJ)
 .PHONY: $(OBJDIR)/%.lib $(OBJDIR)/%.dll
 
 # Check for Makefile in win32 plugins directory otherwise use default Makefile
-plugin-makefile = $(firstword $(realpath \
-					$(PLATDIR)/win32/plugins/$(*F)/Makefile.plugin \
-					../common/Makefile.plugin))
+plugin-makefile = $(realpath $(firstword $(wildcard $(WIN32PLUGINSDIR)/$(1)/Makefile.plugin ../common/Makefile.plugin)))

```

Comment:
👍 should probably be back-ported to win32/win64 squeak/newspeak so as to enable build via MSYS